src/main/scala/de/heikoseeberger/reactiveflows/HttpService.scala

```diff
@@ -74,25 +75,25 @@ object HttpService {
       implicit val timeout = flowFacadeTimeout
       path(Segment / "messages") { flowName =>
         get {
-          onSuccess(flowFacade ? GetMessages(flowName)) {
-            case messages: Seq[Flow.Message] @unchecked => complete(messages)
-            case unknownFlow: UnknownFlow               => complete(StatusCodes.NotFound)
+          onComplete((flowFacade ? GetMessages(flowName)).mapTo[Seq[Flow.Message]]) {
+            case Success(messages)       => complete(messages)
+            case Failure(_: UnknownFlow) => complete(StatusCodes.NotFound)
```

Comment:
Receiving `UnknownFlow` happens on the successful path, not as a failure. What has motivated this change?